docs/yaml-monitoring-templates.rst

```diff
@@ -548,6 +548,12 @@ escalateCount:
   :Type: int
   :Default Value: None
 
+optional:
+  :Description: indicating whether this threshold is optional or not
```

Comment:
Description should be a proper sentence or sentences to match the convention used in the other descriptions. It should also impart additional information than the field name. I don't think this current description helps people understand what "optional" means and what the effect of setting it to true or false would be.

Maybe something more like..

> This threshold will be skipped during installation if the type of threshold isn't available on the system when _optional_ is set to true. Installation will fail if the type of threshold isn't available when _optional_ is set to false.